dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/server/handler/IdleEventHandler.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.qos.server.handler;
+
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelDuplexHandler;
+import io.netty.channel.ChannelHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.timeout.IdleStateEvent;
+import org.apache.dubbo.common.logger.Logger;
+import org.apache.dubbo.common.logger.LoggerFactory;
+
+@ChannelHandler.Sharable
```

Comment:
It's not necessary.

Reply:
if you means the `IdleEventHandler` is not necessary, then the `p.addLast(new IdleStateHandler(0, 0, 5 * 60));` in `QosProcessHandler` may be also not necessary.

Reply:
it is kind of chinglish? QAQ hh

Reply:
I mean the annotation `@Sharable`.

Reply:
but why？

Reply:
It's alway create new `TelnetIdleEventHandler` for channel.

Reply:
Ok, you are right.